src/core/storage/PageCache.cpp

```diff
@@ -0,0 +1,63 @@
+#include <algorithm>
```

Comment:
You don't use any functions from that library

Reply:
:+1: